slsim/lens.py

```diff
@@ -409,14 +385,18 @@ def los_linear_distortions(self):
         return self._los_linear_distortions_cache
 
     def _calculate_los_linear_distortions(self):
-        """Line-of-sight distortions in shear and convergence.
+        """Line-of-sight distortions in shear and convergence for each source.
 
         :return: kappa, gamma1, gamma2
```

Comment:
do you want to specify that these are lists now?

Reply:
I changed it back to original one.